avocado/plugins/list.py

```diff
@@ -276,31 +240,21 @@ def configure(self, parser):
     def run(self, config):
         verbose = config.get('core.verbose')
         write_to_json_file = config.get('list.write_to_json_file')
-        resolver = config.get('list.resolver')
-        runner = 'nrunner' if resolver else 'runner'
         config['run.ignore_missing_references'] = True
-        config['run.test_runner'] = runner
+        config['run.test_runner'] = 'nrunner'
         try:
-            if not resolver:
-                try:
-                    loader.loader.load_plugins(config)
-                    loader.loader.get_extra_listing()
-                except loader.LoaderError as error:
-                    LOG_UI.error(error)
-                    return exit_codes.AVOCADO_FAIL
             suite = TestSuite.from_config(config)
-            if runner == 'nrunner':
-                matrix = self._get_resolution_matrix(suite)
-                self._display(suite, matrix)
+            matrix = self._get_resolution_matrix(suite)
+            self._display(suite, matrix)
 
-                directory = config.get('list.recipes.write_to_directory')
-                if directory is not None:
-                    self.save_recipes(suite, directory, len(matrix))
-            else:
-                matrix = self._get_test_matrix(suite)
-                self._display(suite, matrix)
+            directory = config.get('list.recipes.write_to_directory')
+            if directory is not None:
+                self.save_recipes(suite, directory, len(matrix))
             if write_to_json_file:
                 self._save_to_json(matrix, write_to_json_file, verbose)
         except KeyboardInterrupt:
             LOG_UI.error('Command interrupted by user...')
             return exit_codes.AVOCADO_FAIL
+        except loader.LoaderError as error:
```

Comment:
When you removed the loader from this method, you shouldn't get this error. Sorry that I didn't mention in my first review.

Reply:
No worries, I also found another one. Fixed.